packages/market_repository/pubspec.yaml

```diff
@@ -13,7 +13,7 @@ dependencies:
   market_client:
     git:
       url: https://github.com/SlayerOrnstein/market_client.git
-      ref: v1.1.1
+      ref: 2417521468a17f0dc3cf7ed5b2d4265bff15efaf
```

Comment:
back to a hash instead of a release version?

Reply:
GitHub actions isn't running the action to release for market_client package